.travis.yml

```diff
@@ -10,6 +10,9 @@ matrix:
       env: PERCY_ENABLE=0
 script:
   - npm test
-  - cd packages/react-percy-storybook && yarn link
-  - cd ../../integration-tests && yarn && yarn link @percy-io/react-percy-storybook
-  - npm run storybook:percy
+  - >
```

Comment:
Nice. With this amount of directory state stuff you could potentially throw this into a file like `utils/integration-tests.sh` and just run that here.